examples/README.MD

```diff
@@ -0,0 +1,353 @@
+
+Dataset: N/A
+</td>
+</tr>
+<tr>
+<td>
+
+[moon_example](moon_example)
+</td>
+<td>
+An illustrative implementation using the global non-IID FL method MOON is housed in this folder. On the server-side, very little change is required. On the client-side, the MoonClient base class handles the required training modifications, provided the correct model type.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[mr_mtl_example](mr_mtl_example)
+</td>
+<td>
+This example provides an implementation using the MR-MTL pFL approach. As with other examples, the server-side implementation requires little modification, but the client-side training is modified in the appropriate parent class.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[nnunet_example](nnunet_example)
+</td>
+<td>
+nnUNet is a common and effective library for training centralized segmentation models, especially in clinical settings. This example demonstrates how the library can leverage the nnUNet training approach in a federated setting to train good models in a decentralized setting.
+
+Dataset: MSD Dataset: Task04_Hippocampus
+</td>
+</tr>
+<tr>
+<td>
+
+[perfcl_example](perfcl_example)
+</td>
+<td>
+A set of code demonstrating the use of PerFCL to federally training personalized models is applied. PerFCL is a personalized extension of MOON and is essentially a loss constrained version of FENDA-FL.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[scaffold_example](scaffold_example)
+</td>
+<td>
+This folder provides an example of using the global SCAFFOLD method to federally train models. The approach requires use of specialized server and client classes to facilitate the necessary gradient modifications and information exchanges/aggregation.
+
+Dataset: MNIST
+</td>
+</tr>
+<tr>
+<td>
+
+[sparse_tensor_partial_exchange_example](sparse_tensor_partial_exchange_example)
+</td>
+<td>
+This is an example demonstrating how to use the sparse tensor exchanger implemented in the library. This exchanger allows each client to exchange individual parameters (not just full layers) with the server based on a user specified criteria function. Parameters are aggregated on the server-side using a special form of FedAvg specifically designed to handle heterogeneity in parameters exchanged across clients.
+
+Dataset: CIFAR-10
+</td>
+</tr>
+<tr>
+<td>
+
+[warm_up_example](warm_up_example)
+</td>
+<td>
+This folder provides several implementations of what we refer to as warm-start FL. That is, a model is federally trained using some mechanism, perhaps FedAvg. Thereafter, that pre-trained model is used to initialize further FL training using a different method. The examples herein also provide illustrations of "model surgery" wherein only portions of the warm model are transferred into the new model that will received continued FL training.
```

Comment:
nit: that will *receive continued training

Reply:
👍